### PR TITLE
BrenKt-TRANS-SK1153-WEB-Domain-DMARC

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+**Esta guía explica cómo funciona DMARC y cómo implementarlo para su servicio de correo.**
+
+> [!warning]
+>
+> OVHcloud ofrece servicios cuya configuración, gestión y responsabilidad recaen sobre usted. Por lo tanto, usted deberá asegurarse de que estos funcionen correctamente.
+> 
+> Esta guía le ayudará a realizar las operaciones más habituales. No obstante, si necesita ayuda, le recomendamos que contacte con un [proveedor especializado](https://partner.ovhcloud.com/es-es/directory/). Nosotros no podremos asistirle al respecto. Para más información, consulte la sección [«Más información»](#go-further) de este tutorial.
+>
+> Además, la creación de un registro **DMARC**, independientemente del [servicio de correo de OVHcloud](https://www.ovhcloud.com/fr-ca/emails/), no está disponible en este momento. Esta guía **solo** se aplica a los clientes que tengan una zona DNS activa en OVHcloud para su dominio asociado a una **solución de correo externa**.
+
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md:24]: too many empty lines here

File: pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    - *none*: el propietario del dominio no solicita ninguna acción específica relativa a la entrega de mensajes.
+    - *cuarentena*: si se produce un fallo en la comprobación del mecanismo DMARC, los destinatarios deben considerar el correo como sospechoso. En función de la capacidad del servidor destinatario, esto puede significar «colocar en la carpeta spam» y/o «informar como sospechoso».
+    - *reject*: rechazo de los mensajes de correo electrónico que no superan la comprobación del mecanismo DMARC.
+
+> [!warning]
+>
+> La configuración del parámetro `p=` puede afectar considerablemente a la entrega de los mensajes de correo electrónico del dominio. Es recomendable configurar `p=none` y realizar un análisis de los informes de fallos durante varias semanas para corregir cualquier anomalía. Pasar a `p=cuarentena` o `p=reject` requiere un control total de los parámetros de seguridad del correo electrónico, relativos al [SPF](/pages/web_cloud/domains/dns_zone_spf) y al [DKIM](/pages/web_cloud/domains/dns_zone_dkim). El uso del factor `pct=`, que se muestra a continuación, permite realizar una transición gradual.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md]:
```suggestion
> La configuración del parámetro `p=` puede afectar considerablemente a la entrega de los mensajes de correo electrónico del dominio. Es recomendable configurar `p=none` y realizar un análisis de los informes de fallos durante varias semanas para corregir cualquier anomalía. Pasar a `p=quarantine` o `p=reject` requiere un control total de los parámetros de seguridad del correo electrónico, relativos al [SPF](/pages/web_cloud/domains/dns_zone_spf) y al [DKIM](/pages/web_cloud/domains/dns_zone_dkim). El uso del factor `pct=`, que se muestra a continuación, permite realizar una transición gradual.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+     - **0**: genera un informe de errores de DMARC si todos los mecanismos de autenticación (DKIM **ET** SPF) no pueden producir un resultado "pass" alineado.
+     - **1**: genera un informe de errores de DMARC si un mecanismo de autenticación (DKIM **O** SPF) produce algo distinto de un resultado alineado "success".
+     - **d**: genera un informe de errores de DKIM si se produce un error en el mecanismo de autenticación de DKIM, independientemente de su alineación.
+     - **s**: genera un informe de fallos de SPF si falla el mecanismo de autenticación de SPF, independientemente de su alineación.
+
+- **rf** (valores de texto sin formato separados por comas, el valor predeterminado es `afrf`): esta etiqueta indica el tipo de formato esperado para los informes que proporcionan detalles específicos sobre los errores de autenticación de mensajes. Actualmente solo se admite el `formato de notificación automática de informes (afrf`).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md]:
```suggestion
- **rf** (valores de texto sin formato separados por comas, el valor predeterminado es `afrf`): esta etiqueta indica el tipo de formato esperado para los informes que proporcionan detalles específicos sobre los errores de autenticación de mensajes. Actualmente solo se admite el `afrf` (Auth Failure Reporting Format).
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+![dmarc](images/dns-dmarc-02.png){.thumbnail}
+
+#### Ejemplos de registro <a name="record-example"></a>
+
+> [!warning]
+>
+> En nuestros dos ejemplos, el parámetro `p=`se utiliza en su forma restrictiva para ilustrar el comportamiento de un servicio de correo en este caso.
+>
+> La configuración del parámetro `p=` puede afectar considerablemente a la entrega de los mensajes de correo electrónico del dominio. Se recomienda configurar `p=none` y realizar un análisis de los informes de errores durante varias semanas para corregir cualquier anomalía. Pasar a `p=cuarentena` o `p=reject` requiere un control total de los parámetros de seguridad del correo electrónico, relativos al [SPF](/pages/web_cloud/domains/dns_zone_spf) y al [DKIM](/pages/web_cloud/domains/dns_zone_dkim). El uso del factor `pct=`, que se muestra a continuación, permite realizar una transición gradual.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md]:
```suggestion
> La configuración del parámetro `p=` puede afectar considerablemente a la entrega de los mensajes de correo electrónico del dominio. Se recomienda configurar `p=none` y realizar un análisis de los informes de errores durante varias semanas para corregir cualquier anomalía. Pasar a `p=quarantine` o `p=reject` requiere un control total de los parámetros de seguridad del correo electrónico, relativos al [SPF](/pages/web_cloud/domains/dns_zone_spf) y al [DKIM](/pages/web_cloud/domains/dns_zone_dkim). El uso del factor `pct=`, que se muestra a continuación, permite realizar una transición gradual.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+![dmarc](images/dns-dmarc-04.png){.thumbnail}
+
+Obtenemos el siguiente resultado:
+
+```
+"v=DMARC1; p=cuarentena; pct=100; ruf=mailto:report@mydomain.ovh; fo=0; adkim=r; aspf=s; adkim=r; ri=86400"
+```
+
+- **p=cuarentena**: los mensajes de correo electrónico que no superan las pruebas DMARC se tratan como "sospechosos".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md]:
```suggestion
- **p=quarantine**: los mensajes de correo electrónico que no superan las pruebas DMARC se tratan como "sospechosos".
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+Di seguito trovi la descrizione completa dei tag utilizzati per **la registrazione DMARC** OVHcloud:
+
+- **Versione (v=)**: campo obbligatorio che determina la versione del protocollo DMARC.
+
+- **Regola per il dominio (p=)**: policy che deve essere adottata dal destinatario su richiesta del proprietario del dominio del mittente. Il criterio viene applicato al dominio richiesto e ai sottodomini, a meno che il tag sottodominio **sp=** non indichi istruzioni diverse. I valori possibili sono:
+    - *none* : il proprietario del dominio non richiede alcuna azione specifica riguardante la consegna dei messaggi.
+    - *quarantena*: in caso di fallimento della verifica del meccanismo DMARC, le email devono essere considerate sospette dai destinatari. In base alla capacità del server ricevente, ciò può significare "inserire nella cartella spam" e/o "segnalare come sospetto".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md]:
```suggestion
    - *quarantine*: in caso di fallimento della verifica del meccanismo DMARC, le email devono essere considerate sospette dai destinatari. In base alla capacità del server ricevente, ciò può significare "inserire nella cartella spam" e/o "segnalare come sospetto".
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+    - *reject*: rifiuto delle email che non superano la verifica del meccanismo DMARC.
+
+> [!warning]
+>
+> L’impostazione del parametro `p=` può avere un impatto importante sull’evasione delle email del dominio. È consigliabile configurare `p=none` ed eseguire un'analisi dei rapporti di errore per diverse settimane, per correggere eventuali anomalie. L’upgrade a `p=quarantina` o `p=reject` richiede il pieno controllo delle impostazioni di sicurezza email relative a [SPF](/pages/web_cloud/domains/dns_zone_spf) e [DKIM](/pages/web_cloud/domains/dns_zone_dkim). L’uso del fattore `pct=`, presentato di seguito, consente una transizione graduale.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md]:
```suggestion
> L’impostazione del parametro `p=` può avere un impatto importante sull’evasione delle email del dominio. È consigliabile configurare `p=none` ed eseguire un'analisi dei rapporti di errore per diverse settimane, per correggere eventuali anomalie. L’upgrade a `p=quarantine` o `p=reject` richiede il pieno controllo delle impostazioni di sicurezza email relative a [SPF](/pages/web_cloud/domains/dns_zone_spf) e [DKIM](/pages/web_cloud/domains/dns_zone_dkim). L’uso del fattore `pct=`, presentato di seguito, consente una transizione graduale.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+![dmarc](images/dns-dmarc-02.png){.thumbnail}
+
+#### Esempi di record <a name="record-example"></a>
+
+> [!warning]
+>
+> Nei nostri 2 esempi, il parametro `p=`viene utilizzato nella sua forma restrittiva per illustrare il comportamento di un servizio di posta elettronica in questo caso.
+>
+> L’impostazione del parametro `p=` può avere un impatto importante sull’evasione delle email del dominio. Si consiglia di configurare `p=none` ed eseguire un'analisi dei rapporti di errore per diverse settimane, per correggere eventuali anomalie. L’upgrade a `p=quarantina` o `p=reject` richiede il pieno controllo delle impostazioni di sicurezza email relative a [SPF](/pages/web_cloud/domains/dns_zone_spf) e [DKIM](/pages/web_cloud/domains/dns_zone_dkim). L’uso del fattore `pct=`, presentato di seguito, consente una transizione graduale.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md]:
```suggestion
> L’impostazione del parametro `p=` può avere un impatto importante sull’evasione delle email del dominio. Si consiglia di configurare `p=none` ed eseguire un'analisi dei rapporti di errore per diverse settimane, per correggere eventuali anomalie. L’upgrade a `p=quarantine` o `p=reject` richiede il pieno controllo delle impostazioni di sicurezza email relative a [SPF](/pages/web_cloud/domains/dns_zone_spf) e [DKIM](/pages/web_cloud/domains/dns_zone_dkim). L’uso del fattore `pct=`, presentato di seguito, consente una transizione graduale.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+- **pct=100**: la politica DMARC si applica al 50% dei messaggi provenienti dal flusso di posta elettronica del proprietario del dominio.
+
+- **ruf=mailto:report@mydomain.ovh**: indirizzo e-mail al quale devono essere inviate le notifiche di errore dettagliate utilizzando l'argomento "mailto".
+
+- **fo=0**: opzioni per la generazione di rapporti di errore. Il valore "0" indica che i rapporti di errore DMARC devono essere generati solo se i meccanismi di autenticazione SPF e DKIM non riescono a produrre un risultato allineato "pass".
+
+- **adkim=r**: la modalità di allineamento dell'identificativo DKIM richiesta dal proprietario del dominio è "relaxed" (modalità flessibile). In questa modalità, DKIM deve fornire una firma valida e l'identificativo dell'intestazione "From" può essere parzialmente allineato.
+
+- **aspf=s** - La modalità di allineamento dell'identificativo SPF richiesto è "rigorosa". L'identificativo SPF del dominio allineato deve corrispondere esattamente all'indirizzo IP che invia il messaggio.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.it-it.md]:
```suggestion
- **aspf=s** - La modalità di allineamento dell'identificativo SPF richiesto è "strict". L'identificativo SPF del dominio allineato deve corrispondere esattamente all'indirizzo IP che invia il messaggio.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+
+- **pct=100**: La política DMARC se aplica al 50% de los mensajes procedentes del flujo de correo del propietario del dominio.
+
+- **ruf=mailto:report@mydomain.ovh**: dirección de correo electrónico a la que se enviarán los informes de errores detallados con el argumento "mailto".
+
+- **fo=0**: opciones para generar informes de errores. El valor '0' indica que sólo se deben generar informes de errores DMARC si los mecanismos de autenticación SPF y DKIM no pueden producir un resultado alineado 'pass'.
+
+- **adkim=r**: el modo de alineación de ID de DKIM requerido por el propietario del dominio es "relaxed" (modo flexible). En este modo, DKIM debe proporcionar una firma válida y el identificador del encabezado "From" se puede alinear parcialmente.
+
+- **aspf=s**: el modo de alineación del identificador SPF requerido es "estricto". Esto significa que el SPF del dominio alineado debe coincidir exactamente con la dirección IP remitente del mensaje.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.es-es.md]:
```suggestion
- **aspf=s**: el modo de alineación del identificador SPF requerido es "strict". Esto significa que el SPF del dominio alineado debe coincidir exactamente con la dirección IP remitente del mensaje.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+
+- **Subdomena**: ten wpis musi **musi zaczynać się od** `_dmarc`. Jeśli stosujesz DMARC do całej domeny, nie wpisuj w tym polu niczego innego niż `_dmarc`. Jeśli ustawisz DMARC na subdomenę swojej domeny głównej, dodaj subdomenę po `_dmarc`. Na przykład, jeśli chcesz zastosować DMARC do subdomeny *subdomain.mydomain.ovh*, musisz wpisać `_dmarc.subdomain` w polu „subdomena” dla nazwy domeny *mydomain.ovh*.
+
+Poniżej znajduje się wyczerpujący opis tagów używanych do **rejestracji DMARC** OVHcloud:
+
+- **Wersja (v=)**: obowiązkowe pole określające wersję protokołu DMARC.
+
+- **Reguła dla domeny (p=)**: zasady do przyjęcia przez adresata na wniosek właściciela domeny nadawcy. Polityka dotyczy badanej domeny i subdomen, chyba że znacznik subdomeny **sp=** wskazuje na inne instrukcje. Możliwe wartości to:
+    - *none*: Właściciel domeny nie żąda żadnych konkretnych działań dotyczących dostarczania wiadomości.
+    - *kwarantanna*: jeśli weryfikacja mechanizmu DMARC nie powiedzie się, adresaci muszą uznać e-maile za podejrzane. W zależności od możliwości serwera docelowego może to oznaczać "umieszczenie w folderze spamu" i/lub "zgłoś jako podejrzanego".
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md]:
```suggestion
    - *quarantine*: jeśli weryfikacja mechanizmu DMARC nie powiedzie się, adresaci muszą uznać e-maile za podejrzane. W zależności od możliwości serwera docelowego może to oznaczać "umieszczenie w folderze spamu" i/lub "zgłoś jako podejrzanego".
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+    - *odrzucenie*: odrzucanie e-maili, które nie powiodły się podczas weryfikacji mechanizmu DMARC.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md]:
```suggestion
    - *reject*: odrzucanie e-maili, które nie powiodły się podczas weryfikacji mechanizmu DMARC.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md [hunk 13]
```diff
@@ -0,0 +1,188 @@
...
+     - **0**: generuje raport o niepowodzeniu DMARC, jeśli wszystkie mechanizmy uwierzytelniania (DKIM **I** SPF) nie mogą wygenerować wyrównanego wyniku "pass".
+     - **1**: Generuje raport o niepowodzeniu DMARC, jeśli mechanizm uwierzytelniania (DKIM **LUB** SPF) wygeneruje coś innego niż wyrównany wynik "success".
+     - **d**: generuje raport o niepowodzeniu DKIM, jeśli mechanizm uwierzytelniania DKIM nie powiedzie się, niezależnie od jego wyrównania.
+     - **s**: generuje raport o niepowodzeniu SPF, jeśli mechanizm uwierzytelniania SPF nie powiedzie się bez względu na jego wyrównanie.
+
+- **rf** (wartości w postaci zwykłego tekstu oddzielone przecinkami, wartością domyślną jest `afrf`): Ten znacznik wskazuje oczekiwany typ formatu raportów, które podają szczegółowe informacje o błędach uwierzytelniania wiadomości. Aktualnie obsługiwany jest tylko `format raportowania` awaryjnego (afrf).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md]:
```suggestion
- **rf** (wartości w postaci zwykłego tekstu oddzielone przecinkami, wartością domyślną jest `afrf`): Ten znacznik wskazuje oczekiwany typ formatu raportów, które podają szczegółowe informacje o błędach uwierzytelniania wiadomości. Obecnie obsługiwany jest tylko format `afrf` (Auth Failure Reporting Format).
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md [hunk 14]
```diff
@@ -0,0 +1,188 @@
...
+
+![DMARC](images/dns-dmarc-02.png){.thumbnail}
+
+#### Przykłady rejestracji <a name="record-example"></a>
+
+> [!warning]
+>
+> W naszych dwóch przykładach parametr `p=`jest używany w formie zawężającej, aby zilustrować zachowanie usługi e-mail w tym przypadku.
+>
+> Konfiguracja parametru `p=` może mieć duży wpływ na dostarczalność e-maili Twojej domeny. Zalecamy skonfigurowanie `p=none` i przeprowadzenie przez kilka tygodni analizy raportów o niepowodzeniach, aby naprawić ewentualne anomalie. Zmiana na `p=kwarantanna` lub `p=odrzucenie` wymaga pełnej kontroli nad ustawieniami bezpieczeństwa e-mail dla [SPF](/pages/web_cloud/domains/dns_zone_spf) i [DKIM](/pages/web_cloud/domains/dns_zone_dkim). Zastosowanie czynnika `pct=`, przedstawionego poniżej, umożliwia stopniowe przechodzenie.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pl-pl.md]:
```suggestion
> Konfiguracja parametru `p=` może mieć duży wpływ na dostarczalność e-maili Twojej domeny. Zalecamy skonfigurowanie `p=none` i przeprowadzenie przez kilka tygodni analizy raportów o niepowodzeniach, aby naprawić ewentualne anomalie. Zmiana na `p=quarantine` lub `p=reject` wymaga pełnej kontroli nad ustawieniami bezpieczeństwa e-mail dla [SPF](/pages/web_cloud/domains/dns_zone_spf) i [DKIM](/pages/web_cloud/domains/dns_zone_dkim). Zastosowanie czynnika `pct=`, przedstawionego poniżej, umożliwia stopniowe przechodzenie.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 15]
```diff
@@ -0,0 +1,188 @@
...
+
+O registo **D**omain-based **M**essage **A**uthentication, **R**eporting, and **C**onformance (DMARC) é um mecanismo de segurança de e-mail. Baseia-se no resultado das verificações [SPF](/pages/web_cloud/domains/dns_zone_spf) e [DKIM](/pages/web_cloud/domains/dns_zone_dkim).
+
+**Saiba como o DMARC funciona e como configurá-lo para o seu serviço de e-mail.**
+
+> [!warning]
+>
+> A OVHcloud oferece serviços cuja configuração, gestão e responsabilidade são da sua responsabilidade. Assim, deverá certificar-se de que estes funcionam corretamente.
+> 
+> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/directory/). Não poderemos proporcionar-lhe assistência técnica. Mais informações na secção [« Vá mais longe »](#go-further) deste tutorial.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]:
```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/directory/). Não poderemos proporcionar-lhe assistência técnica. Mais informações na secção [Quer saber mais?](#go-further) deste tutorial.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 16]
```diff
@@ -0,0 +1,188 @@
...
+>
+> Além disso, a implementação de um registo **DMARC**, independentemente do [serviço de e-mail OVHcloud](https://www.ovhcloud.com/fr-ca/emails/), está atualmente indisponível. Este guia diz **unicamente** respeito aos clientes que possuem uma zona DNS ativa na OVHcloud para o seu nome de domínio associado a uma **solução de e-mail externa**.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]: wrong link

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 17]
```diff
@@ -0,0 +1,188 @@
...
+
+A seguir, poderá encontrar uma descrição completa das tags utilizadas para **o registo DMARC** da OVHcloud:
+
+- **Version (v=)**: campo obrigatório que determina a versão do protocolo DMARC.
+
+- **Regra para o domínio (p=)**: política a adotar pelo destinatário a pedido do proprietário do domínio remetente. A política aplica-se ao domínio inquirido e aos subdomínios, exceto se a marca de subdomínio **sp=** indicar instruções diferentes. Os valores possíveis são os seguintes:
+    - *none* : o proprietário do domínio não pede nenhuma ação específica relativa à entrega das mensagens.
+    - *quarentena*: Se a verificação do mecanismo DMARC falhar, os destinatários devem considerar as mensagens de correio eletrónico como suspeitas. Dependendo das capacidades do servidor destinatário, isto pode significar « colocar na pasta de spam » e/ou « indicar como suspeito ».
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]:
```suggestion
    - *quarentine*: Se a verificação do mecanismo DMARC falhar, os destinatários devem considerar as mensagens de correio eletrónico como suspeitas. Dependendo das capacidades do servidor destinatário, isto pode significar « colocar na pasta de spam » e/ou « indicar como suspeito ».
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 18]
```diff
@@ -0,0 +1,188 @@
...
+    - *reject* : rejeição dos e-mails que falham na verificação do mecanismo DMARC.
+
+> [!warning]
+>
+> A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. É aconselhável configurar `p=none` e efetuar uma análise dos relatórios de falha durante várias semanas, de forma a resolver eventuais anomalias. Passar para `p=quarantina` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]:
```suggestion
> A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. É aconselhável configurar `p=none` e efetuar uma análise dos relatórios de falha durante várias semanas, de forma a resolver eventuais anomalias. Passar para `p=quarantine` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 19]
```diff
@@ -0,0 +1,188 @@
...
+
+Abaixo, poderá encontrar a lista das tags utilizadas para criar um **registo TXT** com os parâmetros DMARC. Esta lista é complementar às tags mencionadas na secção anterior « [Registo DMARC](#dmarc-record) ».
+
+- **adkim** (o padrão é `r`): indica o modo de alinhamento DKIM. Os valores são os seguintes:
+    - `r`(relaxed) para o modo flexível: Os e-mails que falham na autenticação DKIM são marcados como « indesejados » pelo servidor destinatário.
+    - `s`(restrito) para o modo restrito: os e-mails que falham na autenticação DKIM são rejeitados pelo servidor destinatário.
+
+- **ruf** - Endereços aos quais devem ser comunicadas as informações de erro específicas da mensagem, incluindo uma lista de texto simples separada por vírgulas Se esta tag estiver presente, o proprietário do domínio remetente pede aos destinatários que enviem relatórios de falhas detalhados sobre os e-mails que falham na avaliação DMARC de forma específica (ver a tag `fo` abaixo). O formato da mensagem a gerar deve seguir o formato especificado para a etiqueta `rf`. A menção « mailto:» deve preceder o destinatário de e-mail (exemplo: `mailto:address@exemplo.com`).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]:
```suggestion
- **ruf** - Endereços aos quais devem ser comunicadas as informações de erro específicas da mensagem, incluindo uma lista de texto simples separada por vírgulas Se esta tag estiver presente, o proprietário do domínio remetente pede aos destinatários que enviem relatórios de falhas detalhados sobre os e-mails que falham na avaliação DMARC de forma específica (ver a tag `fo` abaixo). O formato da mensagem a gerar deve seguir o formato especificado para a etiqueta `rf`. A menção « mailto:» deve preceder o destinatário de e-mail (exemplo: `mailto:address@example.com`).
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 20]
```diff
@@ -0,0 +1,188 @@
...
+Depois de apresentar a zona DNS, clique no botão `Adicionar uma entrada`{.action} e, a seguir, em « Campos » `TXT`{.action}.
+
+- **Subdomínio**: Esta entrada deve **obrigatoriamente começar por** `_dmarc`. Se você está aplicando seu DMARC ao conjunto do domínio, não insira nada além de `_dmarc` nesta caixa. Se definir o DMARC para um subdomínio associado ao seu domínio principal, adicione o subdomínio após `_dmarc`. Por exemplo, se for necessário aplicar o DMARC a um subdomínio *subdomain.mydomain.ovh*, é necessário introduzir `_dmarc.subdomain` na casa « subdomínio » para o nome de domínio *mydomain.ovh*
+
+Abaixo, poderá encontrar a lista das tags utilizadas para criar um **registo TXT** com os parâmetros DMARC. Esta lista é complementar às tags mencionadas na secção anterior « [Registo DMARC](#dmarc-record) ».
+
+- **adkim** (o padrão é `r`): indica o modo de alinhamento DKIM. Os valores são os seguintes:
+    - `r`(relaxed) para o modo flexível: Os e-mails que falham na autenticação DKIM são marcados como « indesejados » pelo servidor destinatário.
+    - `s`(restrito) para o modo restrito: os e-mails que falham na autenticação DKIM são rejeitados pelo servidor destinatário.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]:
```suggestion
    - `s`(strict) para o modo restrito: os e-mails que falham na autenticação DKIM são rejeitados pelo servidor destinatário.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 21]
```diff
@@ -0,0 +1,188 @@
...
+
+![dmarc](images/dns-dmarc-02.png){.thumbnail}
+
+#### Exemplos de registo <a name="record-example"></a>
+
+> [!warning]
+>
+> Nos nossos 2 exemplos, o parâmetro `p=`é utilizado na sua forma restritiva para ilustrar o comportamento de um serviço de e-mail nesse caso.
+>
+> A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. Aconselhamos que configure `p=none` e efetue uma análise dos relatórios de falhas durante várias semanas, a fim de resolver eventuais anomalias. Passar para `p=quarantina` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]:
```suggestion
> A configuração do parâmetro `p=` pode ter um impacto importante na possibilidade de entrega dos e-mails do seu domínio. Aconselhamos que configure `p=none` e efetue uma análise dos relatórios de falhas durante várias semanas, a fim de resolver eventuais anomalias. Passar para `p=quarantine` ou `p=reject` requer um controlo total dos parâmetros de segurança do e-mail, relativamente ao [SPF](/pages/web_cloud/domains/dns_zone_spf) e ao [DKIM](/pages/web_cloud/domains/dns_zone_dkim). A utilização do fator `pct=`, apresentado abaixo, permite uma transição progressiva.
```

File: pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md [hunk 22]
```diff
@@ -0,0 +1,188 @@
...
+
+![dmarc](images/dns-dmarc-04.png){.thumbnail}
+
+Obtemos o seguinte resultado:
+
+```
+"v=DMARC1; p=quarantine; pct=100; ruf=mailto:report@mydomain.ovh; fo=0; adkim=r; aspf=s; adkim=r; ri=86400"
+```
+
+- **p=quarantina**: As mensagens de correio eletrónico que não passem os testes DMARC são tratadas como « suspeitas ».
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dmarc/guide.pt-pt.md]:
```suggestion
- **p=quarantine**: As mensagens de correio eletrónico que não passem os testes DMARC são tratadas como « suspeitas ».
```

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**